deployment.rb

```diff
@@ -91,6 +91,7 @@ def load_configuration()
     'dynamo_tables_table'         => "#{rack_env}_tables",
     'dynamo_tables_index'         => "channel_id-table_name-index",
     'dynamo_properties_table'     => "#{rack_env}_properties",
+    'throttle_data_apis'          => rack_env == :production,
```

Comment:
This is limited to production because `geocoder_redis_url` is only defined in production (see https://manage.chef.io/ -> Policy -> Environments -> production -> Attributes -> cdo_secrets). I assume that means we aren't even running Redis in other environments but I could be mistaken.